### PR TITLE
NFDIV-2382: Add Welsh translation for the pages

### PR DESCRIPTION
**Pull request by FaisalMoJ**
### Change description ###

Enter a description.

### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/NFDIV-2382

**Before merging a pull request make sure that:**

- [ ] tests have been updated / new tests has been added (if needed)
- [ ] README and other documentation has been updated / added (if needed)

**If this ticket will have any visible impact on users and is not behind a feature toggle, make sure that:**
- [ ] this ticket been reviewed by QA
- [ ] the user story been signed off by the PO

Note that bug fixes, dependency updates and technical tasks do not directly impact the user experience and can be merged without QA and PO review.

### CODE REVIEW

File: src/main/steps/applicant1/check-your-conditional-order-answers/content.ts [hunk 1]
```diff
@@ -54,8 +54,32 @@ const en = ({ isJointApplication, isDivorce, userCase, isApplicant2 }: CommonCon
   },
 });
 
-// @TODO translations
-const cy: typeof en = en;
+const cy = ({ isDivorce, partner, isJointApplication, required }: CommonContent) => ({
```
Comment (lara-mpt) [inline src/main/steps/applicant1/check-your-conditional-order-answers/content.ts:57]:
```suggestion
const cy: typeof en = ({ isDivorce, partner, isJointApplication, required }: CommonContent) => ({
```

File: src/main/steps/applicant1/continue-with-your-application/content.ts [hunk 2]
```diff
@@ -35,8 +35,32 @@ const en = ({ isDivorce, partner, isJointApplication, required }: CommonContent)
   },
 });
 
-// @TODO translations
-const cy = en;
+const cy = ({ isDivorce, partner, isJointApplication, required }: CommonContent) => ({
```
Comment (lara-mpt) [inline src/main/steps/applicant1/continue-with-your-application/content.ts:38]:
```suggestion
const cy: typeof en = ({ isDivorce, partner, isJointApplication, required }: CommonContent) => ({
```

**Review by lara-mpt — APPROVED** [review]
Looks good, just added a small comment to add a typeof check for the Welsh - helps to prevent Welsh being forgotten if the English content is updated in the future.

**Comment by FaisalMoJ** [comment]
> Looks good, just added a small comment to add a typeof check for the Welsh - helps to prevent Welsh being forgotten if the English content is updated in the future.

Ah nice one thanks for the suggestion and approving the PR @lara-mpt

File: src/main/steps/applicant1/check-your-conditional-order-answers/content.ts (hunk 1 above)
Comment (FaisalMoJ) [reply src/main/steps/applicant1/check-your-conditional-order-answers/content.ts]: This was updated by mistake so i reverted the changes

File: src/main/steps/applicant1/continue-with-your-application/content.ts (hunk 2 above)
Comment (FaisalMoJ) [reply src/main/steps/applicant1/continue-with-your-application/content.ts]: Assigned type to the const as suggested by Lara

**Pull request merged**